server/migrations/20210716071454-search-index-previousTitles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const searchDocument = `
+    CREATE OR REPLACE FUNCTION documents_search_trigger() RETURNS trigger AS $$
+    begin
+      new."searchVector" :=
+        setweight(to_tsvector('english', coalesce(new.title, '')),'A') ||
+        setweight(to_tsvector('english', coalesce(array_to_string(new."previousTitles", ' , '),'')),'A') ||
```

Comment:
@tommoor, I had to google search to figure this. It works, but do let me know if this is not the way you would have done.

Reply:
Will do a little more research but gut tells me this is the right way – We should probably set the weights like so:

- Current title: `A`
- Current text: `B`
- Previous titles: `C`

Reply:
Works like a charm. 

![Screenshot from 2021-07-17 11-35-18](https://user-images.githubusercontent.com/40148448/126027608-6c2ce917-4320-4109-904d-38028f4a8e82.png)